src/components/toast/toast.scss

```diff
@@ -0,0 +1,109 @@
+:host {
```

Comment:
Add the freshworks theme scss.

Also, all colours and font weight should be used from the theme please.